deployment/network-operator/values.yaml

```diff
@@ -215,6 +215,16 @@ sriovDevicePlugin:
     - name: hostdev
       vendors: [15b3]
 
+ibKubernetes:
+  deploy: false
+  image: ib-kubernetes
```

Comment:
The image is not pullable:
```docker pull ghcr.io/mellanox/ib-kubernetes:v1.0.1
Error response from daemon: Head "https://ghcr.io/v2/mellanox/ib-kubernetes/manifests/v1.0.1": unauthorized```

Reply:
The image has a private visibility on GHCR. I'm working with Mellanox org admins to change that

Reply:
Resolved. The image is publicly available now